### PR TITLE
Add configurations for immunization

### PR DESCRIPTION
**Pull request by makombe**

### CODE REVIEW

File: configuration/globalproperties/fhir-properties.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<config>
+    <globalProperties>
+        <globalProperty>
+            <property>fhir2.administeringEncounterRoleUuid</property>
+		    <value>240b26f9-dd88-4172-823d-4a8bfeb7841f</value>
```
Comment (ojwanganto) [inline configuration/globalproperties/fhir-properties.xml:5]: do we have this metadata in our setup?

**Comment by ojwanganto** [comment]
Then that's fine. May be you can check if it is already in the initializer
module.

File: configuration/globalproperties/fhir-properties.xml (hunk 1 above)
Comment (jecihjoy) [reply configuration/globalproperties/fhir-properties.xml:5]: Initializer has domains for both encounter types and global properties, and it also supports concepts exported from ocl

**Comment by ojwanganto** [comment]
Thank you

**Comment by makombe** [comment]
@ojwanganto I have test and found out one thing was missing, encounter roles config and included it accordingly. We can now save immunizations.
Screenshot from 2024-01-31 11-25-03

**Review by donaldkibet — APPROVED** [review]

**Comment by ojwanganto** [comment]
Thank you

**Pull request merged**